packages/zent/src/alert/README-en_US.md

```diff
@@ -0,0 +1,159 @@
+## Alert 
+
+Alert, providing clear prompt messages.
+
+### Usage guide
+
+-  The content is as simple as possible to reduce reading barrier.
+-  No more than two buttons for Alert, keeping the logic simple.
+
+### Code demo
+
+:::demo Basic usage
+```jsx
+import { Alert } from 'zent';
+
+ReactDOM.render(
+	<Alert>Alert content.</Alert>
+	, mountNode
+);
+```
+:::
+
+:::demo The conent can be non-strings such as a DOM element.
+```jsx
+import { Alert, Icon } from 'zent';
+
+ReactDOM.render(
+	<Alert>
+		<Icon type="error-circle" />
+		<span>Warning：SMS notification during the transaction will be sent via the "Message Push" function of the Marketing Center.</span>
+		<a href="" onClick={e => e.preventDefault()}>Order now</a>
+	</Alert>
+	, mountNode
+)
+```
+:::
+
+
+:::demo Three styles：`info`, `warning` and `danger`
+```jsx
+import { Alert } from 'zent';
+
+ReactDOM.render(
+	<div className="zent-alert-example">
+		<Alert type="info">This is the default info style.</Alert>
+		<Alert type="warning">This is the warning style.</Alert>
+		<Alert type="danger">This is the danger style.</Alert>
+	</div>
+	, mountNode
+);
+```
+:::
+
+:::demo Two sizes
+```jsx
+import { Alert } from 'zent';
+
+ReactDOM.render(
+	<div className="zent-alert-example">
+		<Alert>This is the normal size.</Alert>
+		<Alert size="large">
+			<p>This is the big size</p>
+			<p>SMS notification during the transaction will be sent via the "Message Push" function of the Marketing Center.</p><br />
```

Comment:
SMS notification of trading will be sent via the "Message Push" function of the Marketing Center.